stable/opsportal/templates/ingress-opsportal.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     traefik.frontend.rule.type: PathPrefixStrip
     ingress.kubernetes.io/auth-secret: ops-portal-htpasswd
```

Comment:
This has changed. We probably need to remove all those labels and use forward auth instead

Reply:
done